relaton/models/orgs.py

```diff
@@ -15,4 +16,4 @@ class Organization:
     name: Union[List[str], str]
```

Comment:
@stefanomunarini can you check that `organization.name` shouldn’t be a `GenericStringValue` too? In [RFC1330.yaml](https://github.com/ietf-tools/relaton-data-rfcs/blob/89b975f64334b74af66858e21f30b250c06a2650/data/RFC1330.yaml#L22-L27) it’s at least a `FormattedString`, and [in RNC](https://github.com/relaton/relaton-models/blob/main/grammars/biblio.rnc#L145) it seems to even have variants (though we don’t need to support them for now if they don’t appear in source data).